web_src/js/features/common-global.js

```diff
@@ -1,5 +1,5 @@
 import $ from 'jquery';
-import 'jquery.are-you-sure';
+import '../vendor/ays.js';
```

Comment:
I would prefer to use `../vendor/jquery.are-you-sure.js` to match its package name.

Reply:
Done